authentication/signup.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="/authentication/auth.css" />
+    <title>Ecommerce - Sign up</title>
+  </head>
+  <body>
+    <section class="header-wrapper">
+      <header class="header">
+        <a href="$" class="header-brand h3">Some Business</a>
+        <nav class="header-nav">
+          <ul>
+            <li><a href="#">Categories</a></li>
+            <li><a href="#">Plans</a></li>
+          </ul>
+        </nav>
+        <div class="header-search-wrapper">
+          <label for="search"><i class="fas fa-search"></i></label>
+          <input type="text" id="search" />
+        </div>
+        <div class="header-ctas">
+          <button class="btn btn-outline-primary header-cart-btn">
+            <a href="#"><i class="fas fa-shopping-cart"></i></a>
+          </button>
+          <button class="btn btn-outline-primary">
+            <a href="#">Log in</a>
+          </button>
+          <button class="btn btn-solid-primary"><a href="#">Sign up</a></button>
+        </div>
+      </header>
+    </section>
+    <main>
+      <section class="auth-box">
+        <h1 class="auth-box-heading h4">Sign up and start learning</h1>
```

Comment:
H1 inside that H4 why not use H4 directly

Reply:
cause this is primary heading i.e. why its h1 and the styling comes from h3 class